dcos/dcos-ui
# feat(ServiceList) replace health Status with service status

Replace the health status with service status on dashboard page, this also update the title of the panel

**How to test:**
Deploy a couple applications
Have them have different status
Go to dashboard page, there you will see something like the image below (_no more health checks, only services status)_)

![screen shot 2017-11-09 at 5 14 28 pm](https://user-images.githubusercontent.com/549394/32637935-7f161d78-c571-11e7-968b-a0031158d5ee.png)


Closes DCOS-19104

**Checklist**
